papers/david_dotson/david_dotson.rst

```diff
@@ -61,7 +61,7 @@ Furthermore, with ever-increasing computational power, it is often necessary to
 
 Managing the volume and complexity of these data is becoming increasingly difficult problem in and of itself; data management is a challenge to scientific reproducibility and ultimately presents a substantial barrier to answering scientific questions.
 One approach to this problem is ``datreant``, a namespace package that provides a Pythonic interface to the filesystem and the data that lives within it.
-By consolidating Python-based post-processing and analysis with the accessing, generation, and storage of raw and intermediate data, ``datrent`` is designed to solve a boring problem so we can focus more attention on interesting ones.
+By consolidating Python-based post-processing and analysis with the accessing, generation, and storage of raw and intermediate data, ``datreant`` is designed to solve a boring problem so we can focus on interesting problems.
```

Comment:
In scientific writing i prefer to be explicit even if it means repetition. It reads a little bit less nice for native speakers but helps non native speakers a lot.